ig.ini

```diff
@@ -1,3 +1,3 @@
 [IG]
 ig = fsh-generated/resources/ImplementationGuide-hl7.fhir.us.mcode.json
-template = #custom-template
+template = fhir.base.template
```

Comment:
Why is the template changing?

Reply:
good catch. this was meant to be a preliminary version for the redesign of biomarkers. I'll change it back.